### PR TITLE
Wait until the qcow image is unlocked by the oVirt engine

### PR DESCRIPTION
**Pull request by badnetmask**
On a very busy oVirt engine (or when there is a considerable latency between the Ansible controller and the API), it's possible that the API will release the QCOW2 upload process, but the disk will remain locked for a while. The API will not allow to create a new VM until the disk is unlocked.

This patch introduces a small delay (30 seconds) which the playbook will keep querying the API until the image is unlocked.

### CODE REVIEW

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for this project! 

I am the bot who triggers "standard-CI" builds for this project.
As a security measure, I will not run automated tests on PRs that are not from white-listed contributors.

In order to allow automated tests to run, please ask one of the project maintainers to review the code and then do one of the following:

1. Type `ci test please` on this PR to trigger automated tests for it.
2. Type `ci add to whitelist` on this PR to trigger automated tests for it and also add you to the contributor white-list so that your future PRs will be tested automatically. ( keep in mind this list might be overwritten if the job XML is refreshed, for permanent whitelisting, please follow #3 option )
3. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the project organization, so you'll be able to run tests for all the organization's projects.

File: tasks/qcow2_image.yml
Comment (mwperina) [inline tasks/qcow2_image.yml:103]: Ondro, is it worth to have retries/delay configurable as role paramater?
@machacekondra

Comment (machacekondra) [reply tasks/qcow2_image.yml:103]: Actually I think this is bug here:

https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/cloud/ovirt/ovirt_disk.py#L388

The condition is wrong. We should wait in case of not lun disk.

Comment (badnetmask) [reply tasks/qcow2_image.yml:103]: So, does that mean that the proper solution is to patch the ovirt_disk module, instead of trying to workaround the problem in this role?

Comment (machacekondra) [reply tasks/qcow2_image.yml:103]: Yeah, I think it would be much better solution.

Comment (badnetmask) [reply tasks/qcow2_image.yml:103]: @machacekondra a colleague hit a similar bug while doing disk operations, so I just want to confirm what is the next step here: will you patch the ovirt_disk module, or am I supposed to present a PR?

Comment (acidorius) [reply tasks/qcow2_image.yml:103]: @machacekondra Probably the same issue was encountered (the difference only in ok status of disk). The process of creating the VM template starting before image upload finishes and it cause "Failed to create Template" in most time (approximately 1 successful template creation per 3 tries). Should this issue be delegated to ovirt_disk module or some patch will be provided?

Comment (machacekondra) [reply tasks/qcow2_image.yml:103]: Sorry for late reply, I've sent fix:

https://github.com/ansible/ansible/pull/55426

I will merge this one as well, and when bump on Ansible 2.8 I will remove it. Thanks!

**Review by machacekondra — APPROVED** [review]

**Pull request merged**
